### DISCUSSION TITLE
Authorized ip ranges change trigger node restarts

### DISCUSSION THREAD

**Discussion by humpalu** [Azure/AKS#4179] [Q&A]
Hi,
When I setup or change the authorized ip ranges configuration on my aks cluster, the Azure restarts all of the nodes and evacuate all of the pods. Why is that happens? Can I avoid it? 
Unfortunately the cluster recreation is not an option right now for make the endpoint private ( planned for the next year )

**Comment by philwelz** [comment]
Hi, this seems not normal behaviour. How you are changing the authorized ip ranges configuration?

**Comment by humpalu** [comment]
I just activated it on the Azure Portal with the list of the ip ranges. It was a bit painful due to some wrong pdb settings.
So I would like to avoid it in the future. Or this node recreation in the nodepools were a one-time change to activate the feature?
